resources/manifests/pod-checkpointer-psp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: pod-checkpointer-restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
```

Comment:
Can't we drop all? Use:
 `    - ALL`

Reply:
I copied from `restricted` policy, which I believe is already restrictive enough. Perhaps we can ask @surajssd

Reply:
This policy does not try to be restrictive while allowing several use cases (like the `restricted` policy that is used by default).

I think we should enable **only** what is needed for pod checkpointer. We can take advantage that this policy is to be used only for the pod checkpointer, and limit to the best extent we can :)

Reply:
Dropping everything will make the pod unusable.

Reply:
I meant: what is the minimum set of syscalls we need for this pod? Do we know?

Reply:
Taking note to create an issue and see if we can improve it. Out of scope for this PR.